app/models/plant_population.rb

```diff
@@ -29,6 +29,13 @@ class PlantPopulation < ActiveRecord::Base
 
   include Filterable
 
+  validates :name,
+            presence: true,
+            allow_blank: true
+
+  validates :canonical_population_name,
+            presence: true
```

Comment:
This one is not required according to the schema. If you think it should be required, then it's ok.